From cf4053bca7fb9e3385c5afa5610fff6831d13d54 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sat, 7 Dec 2024 10:41:31 -0700
Subject: [PATCH] Bump Security String to 2024-12-05

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2024-43097  A-350118416    EoP    High       12, 12L, 13, 14, 15
CVE-2024-43762  A-340239088    EoP    High       12, 12L, 13, 14, 15
CVE-2024-43767  A-352631932    RCE    High       12, 12L, 13, 14, 15
CVE-2024-43768  A-349678452    EoP    High       12, 12L, 13, 14, 15

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2024-43764  A-317048495    EoP    High       13, 14
CVE-2024-43769  A-360807442    EoP    High       13, 14, 15

Change-Id: Ic3a66bcbd77e989baca128e06b00d296da910a8a
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 940b154a67..96f3d3ce7e 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -250,7 +250,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2024-11-05
+    PLATFORM_SECURITY_PATCH := 2024-12-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.40.1

